modules/sandboxed-containers-deleting-operator-deployment-cr.adoc

```diff
@@ -0,0 +1,32 @@
+//Module included in the following assemblies:
+//
+// *uninstalling-sandboxed-containers.adoc
+
+[id="sandboxed-containers-deleting-operator-deployment-cr_{context}"]
+= Deleting {sandboxed-containers-first} resources
+
+To uninstall {sandboxed-containers-first}, you must first delete the {sandboxed-containers-first} custom resource `KataConfig`. This removes and uninstalls the `kata` runtime and its related resources from your cluster.
+
+.Prerequisites
+
+* You have {product-title} {product-version} installed on your cluster.
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have no running pods that use `kata` as the `runtimeClassName`.
+** You have installed the OpenShift CLI (`oc`).
+** You the command-line JSON processor (`jq`) installed.
+** Verify that you have no running pods that use `kata` as the `runtimeClassName` by running the following command:
++
+[source,terminal]
+----
+$ oc get pods -A -o json | jq -r '.items[] | select(.spec.runtimeClassName | test("kata")).metadata.name'
+----
+
+.Procedure
+
+. Delete all pods that use `runtimeClassName` with the value of `kata`. 
+. From the {product-title} web console, select `openshift-sandboxed-containers` from the *Projects* list.
+. Navigate to the *Operators* → *Installed Operators* page.
+. Click *{sandboxed-containers-first}*.
+. Click the *{sandboxed-containers-operator}* tab.
+. Click the scroll-down list in the *Operator Details*, and click *Delete `KataConfig`*.
```

Comment:
If you know of a noun to add after "the *Operator Details*", go for it and add it. It's prolly fine as-is, but reads awkwardly.